packages/@aws-cdk/aws-ecs/test/ec2/integ.capacity-provider.ts

```diff
@@ -11,6 +11,9 @@ const vpc = new ec2.Vpc(stack, 'Vpc', { maxAzs: 2 });
 const cluster = new ecs.Cluster(stack, 'EC2CPCluster', {
   vpc,
   enableFargateCapacityProviders: true,
+  defaultCapacityProviderStrategy: [
+    { capacityProvider: 'EC2', base: 10, weight: 50 },
```

Comment:
This integ test fails to deploy. When I ran
```sh
yarn integ --force ec2/integ.capacity-provider.js
```
I received the following error:
```sh
integ-ec2-capacity-provider: deploying...
[0%] start: Publishing 3a7499485875940a7ccde4a965390adddd0d423a55e3e2ae5cc259f3b68814d0:current_account-current_region
[100%] success: Published 3a7499485875940a7ccde4a965390adddd0d423a55e3e2ae5cc259f3b68814d0:current_account-current_region
integ-ec2-capacity-provider: creating CloudFormation changeset...
integ-ec2-capacity-provider | 0/3 | 3:23:36 PM | UPDATE_IN_PROGRESS   | AWS::CloudFormation::Stack                    | integ-ec2-capacity-provider User Initiated
integ-ec2-capacity-provider | 0/3 | 3:23:42 PM | UPDATE_IN_PROGRESS   | AWS::ECS::ClusterCapacityProviderAssociations | EC2CPCluster/EC2CPCluster (EC2CPCluster4CFED4DD) 
integ-ec2-capacity-provider | 0/3 | 3:23:50 PM | UPDATE_FAILED        | AWS::ECS::ClusterCapacityProviderAssociations | EC2CPCluster/EC2CPCluster (EC2CPCluster4CFED4DD) Resource handler returned message: "Out of retries. Last encountered error was: The specified capacity provider strategy cannot contain a capacity provider that is not associated with the cluster. Associate the capacity provider with the cluster or specify a valid capacity provider and try again. (Service: AmazonECS; Status Code: 400; Error Code: InvalidParameterException; Request ID: 2034a01e-a492-48b1-9024-88ca88b60b94; Proxy: null)" (RequestToken: 69e6a6be-511b-f7ae-d74e-ea0a7e5fd2f8, HandlerErrorCode: InvalidRequest)
integ-ec2-capacity-provider | 0/3 | 3:23:54 PM | UPDATE_ROLLBACK_IN_P | AWS::ECS::ClusterCapacityProviderAssociations | EC2CPCluster/EC2CPCluster (EC2CPCluster4CFED4DD) Rolling back the failed resource only.
integ-ec2-capacity-provider | 1/3 | 3:24:14 PM | UPDATE_ROLLBACK_COMP | AWS::ECS::ClusterCapacityProviderAssociations | EC2CPCluster/EC2CPCluster (EC2CPCluster4CFED4DD) Rollback succeeded for the failed resources.
integ-ec2-capacity-provider | 1/3 | 3:24:15 PM | UPDATE_FAILED        | AWS::CloudFormation::Stack                    | integ-ec2-capacity-provider The following resource(s) failed to update: [EC2CPCluster4CFED4DD]. 

Failed resources:
integ-ec2-capacity-provider | 3:23:50 PM | UPDATE_FAILED        | AWS::ECS::ClusterCapacityProviderAssociations | EC2CPCluster/EC2CPCluster (EC2CPCluster4CFED4DD) Resource handler returned message: 
"Out of retries. Last encountered error was: The specified capacity provider strategy cannot contain a capacity 
provider that is not associated with the cluster. Associate the capacity provider with the cluster or specify a valid 
capacity provider and try again. (Service: AmazonECS; Status Code: 400; Error Code: InvalidParameterException;
Request ID: 2034a01e-a492-48b1-9024-88ca88b60b94; Proxy: null)" 
(RequestToken: 69e6a6be-511b-f7ae-d74e-ea0a7e5fd2f8, HandlerErrorCode: InvalidRequest)
integ-ec2-capacity-provider | 3:23:54 PM | UPDATE_ROLLBACK_IN_P | 
AWS::ECS::ClusterCapacityProviderAssociations | EC2CPCluster/EC2CPCluster (EC2CPCluster4CFED4DD) 
Rolling back the failed resource only.
```

Please update this test so it deploys and uses an autoscaling group capacity provider as part of the cluster's default.